dolphinscheduler-task-plugin/dolphinscheduler-task-sql/src/main/java/org/apache/dolphinscheduler/plugin/task/sql/SqlParameters.java

```diff
@@ -266,7 +266,7 @@ public void dealOutParam(String result) {
             //result only one line
             Map<String, String> firstRow = sqlResult.get(0);
             for (Property info : outProperty) {
-                info.setValue(String.valueOf(firstRow.get(info.getProp())));
+                info.setValue(String.valueOf(firstRow.get(info.getProp().toUpperCase())));
```

Comment:
what about the situtation 
  //if sql return more than one line ?  
do you know where is the col name of result  set to UpperCase , is the loggic right?  if user need result of col a, we change it to col A, that is weired.  for example, select col_a from table_x,  I think we need return col_a instead of COL_A,  where the logic of col_a to COL_A should be modified.